extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -159,6 +166,15 @@ export class DraggableDrawer extends AMP.BaseElement {
       this.containerEl_.insertBefore(spacerEl, this.contentEl_);
       this.contentEl_.appendChild(headerShadowRootEl);
       this.element.classList.add('amp-story-page-attachment-ui-v2');
+
+      // Remove background and backgroundColor from parent and put on content element.
+      const bgColor = computedStyle(this.win, this.element)['backgroundColor'];
+      const bg = computedStyle(this.win, this.element)['background'];
+      setStyles(this.contentEl_, {
+        'backgroundColor': bgColor,
+        'background': bg,
+      });
+      setStyle(this.element, 'background', 'none');
```

Comment:
I wonder if what we really mean here, semantically, is that the page attachment is in its own shadow tree.

For example, is it a problem if I specify `amp-story-page-attachment { border: 10px solid pink }`?  Or other ways to violate our expectations in terms of style?